### ISSUE TITLE
Document return types of API methods

### ISSUE THREAD

**Issue by capnrefsmmat** [cmu-delphi/epidatr#204]
For example, the `pub_covidcast()` documentation [just says it returns a tibble](https://github.com/cmu-delphi/epidatr/blob/ee952c518ac0cf7cc25083b6c54a0b284d2c3448/R/endpoints.R#L697C2-L697C2), with no further information on what's in that tibble. At the least, we should say what each row represents (one observation of one geo_value at one time), and perhaps link to the main API documentation that states what columns there will be and what they contain.

Basically, if a user pulls a data frame from one of these methods, it should be easy to find what's in it. Yes, there's a link to the API docs at the top of most documentation pages, but we should use the sections to make things easy to find; and simple things (like what each row represents) can be put into the documentation directly.

**Comment by nmdefries** [comment]
Maybe good for @tinatownes to work on.
